tests/baselines/reference/checkMergedGlobalUMDSymbol.errors.txt

```diff
@@ -0,0 +1,22 @@
+tests/cases/compiler/global.d.ts(6,16): error TS2403: Subsequent variable declarations must have the same type.  Variable 'THREE' must be of type 'typeof import("tests/cases/compiler/global")', but here has type 'typeof import("tests/cases/compiler/three")'.
+
+
+==== tests/cases/compiler/three.d.ts (0 errors) ====
+    export namespace THREE {
+      export class Vector2 {}
+    }
+    
+==== tests/cases/compiler/global.d.ts (1 errors) ====
+    import * as _three from './three';
+    
+    export as namespace THREE;
+    
+    declare global {
+      export const THREE: typeof _three;
+                   ~~~~~
+!!! error TS2403: Subsequent variable declarations must have the same type.  Variable 'THREE' must be of type 'typeof import("tests/cases/compiler/global")', but here has type 'typeof import("tests/cases/compiler/three")'.

```

Comment:
Is this correct error? Seems very confusing

Reply:
It is - `tests/cases/compiler/global` is the file with the `export as namespace`, while `three` is the `export namespace THREE` file.